### PR TITLE
Capwap reassembly fix

### PR DESCRIPTION
**Pull request by ybrustin**

### CODE REVIEW

**Comment by ybrustin** [comment]
recheck

**Comment by ybrustin** [comment]
recheck

File: scripts/automation/trex_control_plane/interactive/trex/common/services/trex_service_ap.py [hunk 1]
```diff
@@ -362,10 +362,10 @@ def process_capwap_ctrl(self, rx_bytes, ap):
                 capwap_assemble['buf'] += control_str
 
                 if rx_pkt.is_last_fragment():
-                    capwap_assemble['assembled'] = CAPWAP_CTRL(
+                    capwap_assemble['assembled'] = bytes(CAPWAP_CTRL(
```
Comment (zittix) [inline scripts/automation/trex_control_plane/interactive/trex/common/services/trex_service_ap.py:365]: can you fix the same here https://github.com/cisco-system-traffic-generator/trex-core/blob/master/scripts/automation/trex_control_plane/interactive/trex/wireless/trex_wireless_worker.py#L1091 ?

Comment (mamonney) [inline scripts/automation/trex_control_plane/interactive/trex/common/services/trex_service_ap.py:365]: Can you fix the same here https://github.com/cisco-system-traffic-generator/trex-core/blob/master/scripts/automation/trex_control_plane/interactive/trex/wireless/trex_wireless_worker.py#L1091 ?

**Review by mamonney — APPROVED** [review]

**Comment by hhaim** [comment]
rebased.

**Pull request closed without merging**